mystbin/backend/utils/db.py

```diff
@@ -279,15 +289,17 @@ async def put_paste(
             resp["files"] = [dict(file) for file in inserted]
 
             return resp
-    
+
     @wrapped_hook_callback
     async def edit_paste(
         self,
         paste_id: str,
-        pages: List[Dict[str, str]],
+        *,
         author: int,
-        password: Optional[str] = None,
-    ) -> Optional[Dict[str, Union[str, List[Dict[str, str]]]]]:
+        new_expires: Optional[datetime.datetime] = None,
+        new_password: Optional[str] = None,
+        files: Optional[list[payloads.PasteFile]] = None,
+    ) -> Optional[Dict[str, Union[str, int, datetime.datetime]]]:
```

Comment:
does this need to return anything if the endpoint simply returns "OK"?

Reply:
Yeah since this has handling to return an int on failure. The endpoint checks that what we return isn't an int that equals `404`.

I suppose it could be simplified to `Optional[int]` in this case?

Reply:
`Optional[int]` would be better ya